gateway/ln-gateway/src/lib.rs

```diff
@@ -277,18 +277,24 @@ impl Gateway {
                 .expect("Failed to build gateway client"),
         );
 
-        if let Err(e) = self.load_actor(client.clone(), route_hints).await {
-            error!("Failed to connect federation: {}", e);
-        }
-
         if let Err(e) = self.client_builder.save_config(client.config()) {
```

Comment:
We still want to try load actor before persisting the client configuration using `client_builder.save_config`